src/diffusers/pipelines/stable_diffusion/convert_from_ckpt.py

```diff
@@ -1190,16 +1195,40 @@ def download_from_original_stable_diffusion_ckpt(
                     requires_safety_checker=False,
                 )
             else:
-                pipe = StableDiffusionPipeline(
-                    vae=vae,
-                    text_encoder=text_model,
-                    tokenizer=tokenizer,
-                    unet=unet,
-                    scheduler=scheduler,
-                    safety_checker=None,
-                    feature_extractor=None,
-                    requires_safety_checker=False,
-                )
+                if "LatentInpaintDiffusion" in original_config.model.target:
```

Comment:
Sure, I'm okay with that change. My only concern is if a different model in the future (unrelated to inpainting) also uses 8 channels. I'll defer to your knowledge here, since my understanding is limited. Maybe 8 channels always signify inpainting?

The links to the yaml config files are at:
* 1.5-inpainting: https://github.com/runwayml/stable-diffusion/blob/main/configs/stable-diffusion/v1-inpainting-inference.yaml#L3
* 2.0-inpainting: https://github.com/Stability-AI/stablediffusion/blob/main/configs/stable-diffusion/v2-inpainting-inference.yaml#L3

I'm okay with whatever you suggest, happy to make the change. Thanks!

Reply:
8 channels actually kind of imply that the underlying pipeline has another spatial conditioning (can be a real image, can be a mask, etc.).